components/applications-service/pkg/storage/postgres/db.go

```diff
@@ -96,6 +96,7 @@ type service struct {
 	DeploymentID int32  `db:"deployment_id"`
 	SupID        int32  `db:"sup_id"`
 	Channel      string `db:"channel"`
+	FullPkgIdent string `db:"full_pkg_ident"`
```

Comment:
I think we agreed this field should be called `package_ident` or the long name `package_identifier`.

Reply:
oh I thought you meant I should change it from id to ident, thus it became full_pkg_ident. I can change it again though I don't have an attachment to any name.